src/avram/criteria.cr

```diff
@@ -16,24 +16,29 @@ class Avram::Criteria(T, V)
     rows
   end
 
-  def is(value)
-    add_clause(Avram::Where::Equal.new(column, V::Lucky.to_db!(value)))
+  def eq(_value : Nil)
+    {{ raise "Use `nilable_eq` instead of `eq` if trying to check against a nilable value." }}
+  end
+
+  def eq(value)
+    nilable_eq(value)
+  end
+
+  def nilable_eq(_value : Nil)
+    add_clause(Avram::Where::Null.new(column))
     rows
   end
 
-  def not(value) : T
-    is_not(value)
+  def nilable_eq(value)
+    add_clause(Avram::Where::Equal.new(column, V::Lucky.to_db!(value)))
```

Comment:
I think this might make more sense with the method body swapped with `eq`. So this method body would replace `eq` and this method (`nilable_eq(value)`) would call `eq(value)`. Does that make sense?